tests/main/upgrade-from-release/task.yaml

```diff
@@ -38,6 +38,7 @@ execute: |
         apt install -y snapd=2.48.3
     fi
     declare -A EXPECTED_SNAPD_VERSIONS=(
+        ["22.10"]='2.55.3\+22.04'
```

Comment:
Will this work? Right now we have `snapd | 2.57.1+22.10            | kinetic` but it's a moving target so maybe we should just skip this on kinetic until it's stable? Or is there a trick played in the image where we have a jammy source in the sources.list so that we can always get the `2.55.3\+22.04` version? if so there should be a comment in there too.

Reply:
done